tests/e2e/specs/themes/themes.test.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	installTheme,
+	deleteTheme,
+	isThemeInstalled,
```

Comment:
Just a reminder that we should probably test these individually here in core instead, and rewrite the utils using rest API or other alternatives.

Reply:
Do you mean we should rewrite these utilities in Core @kevin940726?

Reply:
I mean we should test these utilities in core, so that we don't have to do them manually in `e2e-test-utils`. Currently, these utils are implemented in a _slow_ way of manually navigating to the themes page and interact with the buttons. We should be able to rewrite them using programmatic methods like REST API so that they can be faster. We still have to write e2e tests for installing and deleting themes though and they should be in core rather than in gutenberg.

Reply:
That makes sense.

The `isThemeInstalled` utility would benefit to be rewritten using the REST API.
Not sure if we can delete and install themes using the API though 🤔.

I will go ahead and create a new PR with a utility folder and functions.